bait-files/src/main/webapp/submission.js

```diff
@@ -0,0 +1,10 @@
+function fetchBlobstoreUrlAndShowImage() {
```

Comment:
Didn't we decide that his method should be just called fetchBlobstoreUrl?  (since it doesn't show the image)

Reply:
Same as above but should be good now! not sure what was happening earlier but I just pushed and now it's showing all of the changes I made - thanks for pointing that out!